examples/paywall-tester/src/main/java/com/revenuecat/paywallstester/PaywallTesterApp.kt

```diff
@@ -46,6 +47,12 @@ private fun AppNavHost(
                             .plus("?${PaywallScreenViewModel.FOOTER_CONDENSED_KEY}=true"),
                     )
                 },
+                navigateToPaywallByPlacementScreen = { placementId ->
```

Comment:
While I'm ok leaving this in paywall tester, I wonder if it would make more sense to add this to purchase tester... I guess ideally we would unify both, but that might be a bigger effort 😅

Reply:
Ah yeah yeah, I can also add to purchase tester! It's just more fun to show paywalls but totally makes sense there too 😛